spec/models/comment_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Validations For the Comment model' do
+    before(:each) do
+      @user = User.new(name: 'Daniel', photo: 'image_link', bio: 'Developer from Nigeria')
+      @post = Post.new(author: @user, title: 'My title', text: 'My text')
+      @comment = Comment.new(text: 'First comment', author_id: @user.id, post_id: @post.id)
+    end
+
+    it 'if title is present' do
+      @comment.text = nil
+      expect(@comment).to_not be_valid
+    end
+
+    it 'if author_id is integer' do
+      @comment.author_id = 'string'
+      expect(@comment).to_not be_valid
+    end
+
+    it 'if post_id is integer' do
+      @comment.post_id = 'string'
+      expect(@comment).to_not be_valid
+    end
+  end
+end
```

Comment:
- [ ] You should add a test for the `update_comments_counter` method. Kindly add it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec_n%2B1.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.)